src/mcp_server/observability_mcp.py

```diff
@@ -85,3 +89,6 @@ def _register_mcp_tools(self) -> None:
         self.mcp.tool()(query_tempo_tool)
         self.mcp.tool()(get_trace_details_tool)
         self.mcp.tool()(chat_tempo_tool)
+        # Register Korrel8r tools
+        self.mcp.tool()(korrel8r_query_objects)
```

Comment:
I think we can register it if `korrel8r` enabled ?

Reply:
Sounds good. Done!